test/ActiveLogin.Authentication.BankId.AspNetCore.Test/BankId_Authentication_Tests.cs

```diff
@@ -161,7 +166,60 @@ public async Task BankIdAuthentication_Login_Returns_Form()
             Assert.Contains("<form id=\"bankIdLoginForm\">", content);
         }
 
+        [Fact]
+        public async Task AutoLaunch_Sets_Correct_RedirectUri()
+        {
+	        // Arrange mocks
+	        var identityNumber = "195008212226";
```

Comment:
Nice that you use a test PIN :)
My site says it's valid to use in code ;)

https://swedish.identityinfo.net/personalidentitynumber/validate?number=195008212226

Reply:
Yes, this is from skatteverket open data since using "real" numbers would cause all kinds of issues with privacy laws.

Reply:
Exactly :) The site I linked to is absed on that dataset. I created it bacause so many people are using real PINs in their code...